### ISSUE TITLE
tincr::cache::directory_path not found

### ISSUE THREAD

**Issue by nelsobe** [bradselw/tincr#13]
I have done a Team->Merge with official/master.  It shows no changes.  Still have the problem.   Below are the contents of my original email:

---

I have been trying to generate an XDLRC file for an Artix7 part.    When I run, it gets to:

Percent complete: 100%

and then throws this error message:

```
invalid command name “::tincr::cache::directory_path”
```

I have looked and that routine is in the code.  However, none of the commands in that file are executable.  I tried a:

```
package require tincr.cad.cache
```

it returned a 0.0 and then the commands were available.  So, I believe it is a package inclusion problem.

**Comment by bradselw** [comment]
This may be a documentation issue. I checked your fork and saw that it was missing the fix for bug #10, so I went ahead and created a pull request on [nelsobe/tincr](https://github.com/nelsobe/tincr/pull/1). Go there and complete the pull request, then from Eclipse do Team->Pull.

**Comment by bradselw** [comment]
Worked out this issue with nelsobe offline. Closing.
